src/Bicep.Core/SourceCode/SourceArchive.cs

```diff
@@ -121,8 +121,14 @@ public static Stream PackSourcesIntoStream(SourceFileGrouping sourceFileGrouping
             return PackSourcesIntoStream(sourceFileGrouping.EntryFileUri, documentLinks, sourceFileGrouping.SourceFiles.ToArray());
         }
 
+        public static Stream PackSourcesIntoStream(Uri entrypointFileUri, params ISourceFile[] sourceFiles)
+        {
+            throw new Exception("asdfg");
+            //return PackSourcesIntoStream(entrypointFileUri, documentLinks: null, sourceFiles);
```

Comment:
This looks like an oversight, I'm guessing it got missed in the midst of other changes

Reply:
Accidentally checked in